notebooks/eda/related_links_eda_all_data.ipynb

```diff
@@ -0,0 +1,620 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 10,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os \n",
+    "import pandas as pd\n",
+    "import numpy as np\n",
+    "from ast import literal_eval\n",
+    "from collections import Counter\n",
+    "from operator import itemgetter"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 1. Read in data/drop columns to reduce size"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "DATA_DIR = os.getenv(\"DATA_DIR\")\n",
+    "filename = \"preprocessed_taxon_pageseq_20190114_20190116.csv.gz\"\n",
+    "df_file = os.path.join(DATA_DIR, \"processed_journey\", filename)\n",
+    "df_reduced_file = os.path.join(DATA_DIR, \"processed_journey\", \"reduced_\"+filename)\n",
+    "# df_rel_file = os.path.join(DATA_DIR, \"processed_journey\", \"rel_\"+filename)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Load up unfiltered data: 14-16/01/19, one off run"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Original dataset shape (5048130, 17)"
```

Comment:
When i read in and ran shape on the df, I got (5048130, 11)

Reply:
Yes, I'm assuming you're using the reduced dataframe file, not original.

Reply:
ah, yes